time/unit.go

```diff
@@ -56,6 +56,20 @@ func (tu Unit) Value() (time.Duration, error) {
 	return 0, errUnrecognizedTimeUnit
 }
 
+// Count returns the number of units contained within the duration.
+func (tu Unit) Count(d time.Duration) int {
+	if d < 0 {
+		return -1
```

Comment:
Hm why not return an (int, error)? Returning -1 seems weird.

Reply:
I did that originally, but as Prateek pointed out it's a little weird for a function as simple as Count to return an error. It's kind of like `bytes.Index`, is the value not being present really worth an error? I don't feel too strongly though, wouldn't mind going back to the original way.

Reply:
Well I think it's unconventional for `Count` to return -1 when the input is invalid. Why not have a `Count` return `(int, error)` and a `MustCount` return `int` and panics if the input is invalid? This way the caller can always use `MustCount` if input is known to be valid. This is the same approach adopted in other repos as well as in many golang packages (`Compile` and `MustCompile` in the `regexp` package)

Reply:
Sounds good to me, in fact I like the `Count` and `MustCount` approach better than just `Count` with either signature.